### PR TITLE
Fix: Message shield alignment is not right.

### PR DESCRIPTION
**Pull request by manancodes**
## Checklist

- [x] Tests written for new code (and old code if feasible)
- [x] Linter and other CI checks pass
- [x] Sign-off given on the changes (see [CONTRIBUTING.md] (https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

Fixes: https://github.com/vector-im/element-web/issues/26142
Signed-off-by: Manan Sadana manancodes.dev@gmail.com

This was the behaviour before the fix:
Screenshot 2023-10-03 at 3 49 07 PM

And this is the behaviour after the fix:
Screenshot 2023-10-06 at 12 27 06 AM
Screenshot 2023-10-06 at 12 26 32 AM

Type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix: Message shield alignment is not right. ([\#11703](https://github.com/matrix-org/matrix-react-sdk/pull/11703)). Fixes vector-im/element-web#26142. Contributed by @manancodes.

### CODE REVIEW

File: res/css/views/rooms/_EventTile.pcss [hunk 1]
```diff
@@ -822,6 +822,9 @@ $left-gutter: 64px;
     }
 }
 
+.mx_EventTile_e2eIcon_warning {
+    align-self: center;
```
Comment (germain-gg) [inline res/css/views/rooms/_EventTile.pcss:826]: i dont think this is the behgaviour we want, the shield should be aligned with the first line of text, but not aligned with the center when a block goes over multiple lines

**Review by germain-gg — CHANGES_REQUESTED** [review]

File: res/css/views/rooms/_EventTile.pcss (hunk 1 above)
Comment (manancodes) [reply res/css/views/rooms/_EventTile.pcss:826]: Oh ok. I will fix this.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Tested on the deployment. It does not look aligned :frowning_face: Can you have another look? Also be sure to test with different font sizes.

image

**Comment by manancodes** [comment]
Sure. I will look into this.

**Comment by manancodes** [comment]
How do I get the grey shield in messages? I don't see it in a verified session.
Screenshot 2023-10-11 at 10 51 31 PM

**Comment by manancodes** [comment]
It is really hard to align it on all font sizes, specially on smallest font size. Any ideas?

**Comment by manancodes** [comment]
Made changes to align it properly on all layouts and font sizes.

**Review by germain-gg — APPROVED** [review]
Looks about right! Thank you very much for persevering here

**Review by weeman1337 — APPROVED** [review]
Looks good now, thank you :hugs:

**Pull request merged**